planner/core/integration_test.go

```diff
@@ -413,8 +415,12 @@ func (s *testIntegrationSerialSuite) TestBroadcastJoin(c *C) {
 	_, err := tk.Exec("explain select /*+ broadcast_join(fact_t, d1_t), broadcast_join_local(d1_t) */ count(*) from fact_t left join d1_t on fact_t.d1_k = d1_t.d1_k")
 	c.Assert(err, NotNil)
 	c.Assert(err.Error(), Equals, "[planner:1815]Internal : Can't find a proper physical plan for this query")
-	// join with non-equal condition not supported
-	_, err = tk.Exec("explain select /*+ broadcast_join(fact_t, d1_t) */ count(*) from fact_t join d1_t on fact_t.d1_k = d1_t.d1_k and fact_t.col1 > d1_t.value")
+	// nullEQ not supported
```

Comment:
`NULL` should be considered in out joins. so for inner joins, we may ignore it?

Reply:
`nullEQ` there means join using `a <=> b` as the equal condition, TiFlash only support `a = b` as the equal condition, so all types of joins is disabled if the join equal condition is `nullEQ`